src/application.ts

```diff
@@ -2,17 +2,20 @@
 // Node module: loopback-next-hello-world
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
-import {Application} from '@loopback/core';
-import {AuthenticationComponent, AuthenticationBindings} from '@loopback/authentication';
-import {validateApiSpec} from '@loopback/testlab'
-import {MyAuthStrategyProvider} from './providers/auth-strategy';
-import {HelloWorldController} from './controllers/hello-world';
-import {MySequence} from './sequence';
+import { Application } from '@loopback/core';
```

Comment:
coding style - no spaces inside/around curly braces

Reply:
just curious.. we don't have linting anymore?